directus/plural/recipes/directus-azure.yaml

```diff
@@ -19,6 +19,10 @@ sections:
     - name: hostname
       documentation: FQDN to use for your directus installation
       type: DOMAIN
+    - name: directusBucket
+      documentation: the bucket for blob assets stored in directus, like images
+      type: BUCKET
+      default: directus
```

Comment:
This will not be used as we don't handle blob storage for now and can confuse people running directus on azure